src/domain/enrolment/utils.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  EnrolOccurrenceMutationInput,
+  Language,
+  StudyLevel,
+  NotificationType,
+} from '../../generated/graphql';
+import { EnrolmentFormFields } from './enrolmentForm/EnrolmentForm';
+
+const getNotificationType = (
+  values: EnrolmentFormFields
+): NotificationType | undefined => {
+  if (values.hasEmailNotification && values.hasSmsNotification)
+    return NotificationType.EmailSms;
+  else if (values.hasEmailNotification) return NotificationType.Email;
+  else if (values.hasSmsNotification) return NotificationType.Sms;
+
+  return undefined;
```

Comment:
I think returning null here would be better practise. functions return undefined implicitly anyway